DoRunDoRun/Sources/Features/Running/RunningReady/RunningReadyView.swift

```diff
@@ -0,0 +1,96 @@
+import SwiftUI
+import ComposableArchitecture
+import NMapsMap
+
+/// 러닝 시작 전 친구들의 위치와 상태를 표시하는 메인 화면(View)
+///
+/// `RunningReadyFeature`의 상태를 기반으로
+/// 지도(`RunningReadyMapView`), 친구 현황 시트(`FriendStatusSheet`),
+/// 그리고 "오늘의 러닝 시작" 버튼으로 구성됩니다.
+struct RunningReadyView: View {
+    let store: StoreOf<RunningReadyFeature>         // TCA Store (친구 정보, 포커스 상태 등 관리)
```

Comment:
```suggestion
    @Bindable var store: StoreOf<RunningReadyFeature>         // TCA Store (친구 정보, 포커스 상태 등 관리)
```
@State를 store 내부로 옮겨서 바인딩해서 사용 가능해여